fineract-core/src/main/java/org/apache/fineract/portfolio/savings/data/SavingsAccountTransactionData.java

```diff
@@ -782,7 +783,8 @@ public static SavingsAccountTransactionData withWithDrawalTransactionDetails(
                 savingsAccountTransactionData.runningBalance, savingsAccountTransactionData.reversed,
                 savingsAccountTransactionData.transfer, savingsAccountTransactionData.paymentTypeOptions,
                 savingsAccountTransactionData.interestedPostedAsOn, savingsAccountTransactionData.submittedByUsername,
-                savingsAccountTransactionData.note, savingsAccountTransactionData.lienTransaction);
+                savingsAccountTransactionData.note, savingsAccountTransactionData.lienTransaction,
+                savingsAccountTransactionData.getSubmittedOnDate());
```

Comment:
if all the other fields are referenced directly instead of calling the getter, i think it's okay to follow that pattern: `savingsAccountTransactionData.submittedOnDate`

Reply:
I agree with you that having the same pattern for all the parameters would be the best. But it is not a good pattern to call the fields directly, so I am not convinced that it is better, to follow a wrong pattern than break it.

Reply:
I think we can modify all other fields to use the getter, which would be the better way to do it.